src/gz/bufread.rs

```diff
@@ -167,12 +167,22 @@ impl<R: BufRead + Write> Write for GzEncoder<R> {
     }
 }
 
-/// A gzip streaming decoder
+/// A decoder for the first member of a [gzip file].
```

Comment:
Since GzDecoder is used for formats that provide an index to lookup a single member that may not be the first, I'd prefer this to be
```suggestion
/// A decoder for a single member of a [gzip file].
```